server/replicator_test.go

```diff
@@ -280,8 +282,9 @@ func TestCommitOnISRShrink(t *testing.T) {
 
 	// Kill a stream follower.
 	leader := getPartitionLeader(t, 10*time.Second, name, 0, servers...)
+
 	var follower *Server
-	for i, server := range servers {
+	for i, server := range lateJoiner {
```

Comment:
What if the late joiner server is the leader? In this case, `follower` will not get set and be `nil`?

Reply:
Fixed. `lateJoiner` now has at least 2 server, so there will be always one follower in this group